test/configCases/url/publicPath/webpack.config.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+	mode: "development",
+	output: {
+		publicPath: "assets/",
+		urlModuleFilename: "file.[ext]"
```

Comment:
```suggestion
		urlModuleFilename: "file[ext]"
```

Reply:
`[ext]` includes the dot. Otherwise it become weird when the original file has no extension at all.